code/modules/goonchat/browserassets/css/browserOutput.css

```diff
@@ -393,13 +406,13 @@ h1.alert, h2.alert		{color: #000000;}
 .big					{font-size: 24px;}
 .reallybig				{font-size: 32px;}
 .extremelybig			{font-size: 40px;}
-.greentext				{color: #00FF00;	font-size: 24px;}
-.redtext				{color: #FF0000;	font-size: 24px;}
+.greentext				{color: #059223;	font-size: 24px;}
+.redtext				{color: #c51e1e;	font-size: 24px;}
```

Comment:
Why the fuck does he make the green & redtext not green and red? What the hell is *wrong* with him?

Reply:
Might be fixed in one of the fixes